R2API/R2API.cs

```diff
@@ -78,7 +78,6 @@ public class R2API : BaseUnityPlugin {
 
             // Make sure that modded dedicated servers are recognizable from the server browser
             On.RoR2.SteamworksServerManager.UpdateHostName += (orig, self, hostname) => {
-                orig(self, $"[MOD] {hostname}");
```

Comment:
Shouldn't this be `orig(self, hostname)`?

Reply:
Yes I think actually we want to remove this hook

Reply:
The entire hook should have been removed, can't create An issue for it right now